services/QuillLMS/app/controllers/profiles_controller.rb

```diff
@@ -87,34 +87,44 @@ def staff
     render :staff
   end
 
-  private def student_exact_scores(user, unit_activity_params)
-    activity_id = unit_activity_params['activity_id']
-    classroom_unit_id = unit_activity_params['classroom_unit_id']
-    ua_id = unit_activity_params['ua_id']
-
-    user_id = user.id
-    cache_key = "#{Student::EXACT_SCORES_CACHE_KEY}/#{user_id}/#{activity_id}/#{classroom_unit_id}"
-
-    Rails.cache.fetch(cache_key, expires_in: 8.hours) do
-      activity_sessions = ActivitySession
-        .includes(:concept_results, :activity, :unit)
-        .where(
-          user_id:,
-          activity_id:,
-          classroom_unit_id:,
-          state: ActivitySession::STATE_FINISHED
-        )
-
-      {
-        'sessions' => activity_sessions.map { |as| format_activity_session_for_tooltip(as, user) },
-        'completed_attempts' => activity_sessions.length,
-        'activity_id' => activity_id,
-        'classroom_unit_id' => classroom_unit_id,
-        'ua_id' => ua_id
-      }
+  ACTIVITY_ID = 'activity_id'
+  CLASSROOM_UNIT_ID = 'classroom_unit_id'
+  UA_ID = 'ua_id'
```

Comment:
I feel like we usually group constants at the top of a file. I'm not sure it really matters either way because I'm normally cmd+f-ing anyway, but we might want to move these for consistency.

Reply:
That's fair. I've starting putting constants close to their use, but I think that breaks from the pattern we use everywhere else, so I'll move them to the top.